include.js

```diff
@@ -1978,6 +1978,13 @@ let cards = [
     author: 'Ishan', 
     githubLink: 'https://github.com/ishanchaskar' 
   },
+  {
+    artName: 'Circular multi-color Disk',
+    pageLink: './Art/ReneDev01/index.html', 
+    imageLink: './Art/ReneDev01/Disk-2023-10-18-09-38-09', 
```

Comment:
# Feedback - incorrect `imageLink`
- [ ] Please add the `.gif` extension to correctly refer to your  animated file
## Context
```js
 imageLink: './Art/ReneDev01/Disk-2023-10-18-09-38-09', 
```

Reply:
I correct it.